context.go

```diff
@@ -6,6 +6,12 @@ import (
 	"time"
 )
 
+// HeightView is a block height/consensus view pair.
+type HeightView struct {
```

Comment:
We need to adjust README wrt these changes.

Reply:
Fixed.